error.vue

```diff
@@ -0,0 +1,127 @@
+<template>
+  <div>
+    <qiskit-ui-shell variant="hide-account" @on-click="onClick" />
+    <div class="error-page">
+      <div class="cds--grid">
+        <div class="cds--row">
+          <div class="cds--col-lg-4">
+            <h1 class="error-page__title">{{ props.error?.statusCode }}</h1>
+            <p>
+              {{ props.error?.message }}
+            </p>
+            <p class="error-page__message">
+              Or as we like to say,
+              <!-- eslint-disable vue/no-v-html -->
+              <span v-html="randomMessage" />
+              <!-- eslint-enable -->
+            </p>
+
+            <UiCta
+              :label="BackToHomeCTA.label"
+              :segment="BackToHomeCTA.segment"
+              :url="BackToHomeCTA.url"
```

Comment:
We usually camel-case the variables:

```suggestion
              :label="backToHomeCta.label"
              :segment="backToHomeCta.segment"
              :url="backToHomeCta.url"
```